auth.go

```diff
@@ -411,10 +413,16 @@ func (x *Central) Login(identity, password string) (sessionkey string, token *To
 		err = authErr
 		x.Stats.IncrementInvalidPasswords(x.Log)
 		x.Log.Infof("Login Authentication failed (%v) (%v)", identity, err)
+
+		// Send failed login to audit service
+		auditUserAction(identity, password, "Login Authentication failed")
 		return sessionkey, token, err
 	}
 	x.Log.Infof("Login authentication success (%v)", userId)
 
+	// Send success login to audit service
+	auditUserAction(identity, password, "Login Authentication success")
```

Comment:
The error handling for auditUserAction should live inside auditUserAction, because the caller's behaviour doesn't change if the audit calls succeeds or fails.
I would remove those comments, since they're not adding any new information